edx_arch_experiments/kafka_consumer/signals/receivers.py

```diff
@@ -0,0 +1,28 @@
+"""Signal receivers for the event bus"""
+
+import logging
+
+import attrs
+from django.dispatch import receiver
+from openedx_events.enterprise.data import SubscriptionLicenseData
+from openedx_events.enterprise.signals import SUBSCRIPTION_LICENSE_MODIFIED
+
+logger = logging.getLogger(__name__)
+
+
+@receiver(SUBSCRIPTION_LICENSE_MODIFIED)
+def log_event_from_event_bus(**kwargs):  # pragma: no cover
+    """
+    Log event received and transmitted from event bus consumer
+
+    :param kwargs: event data sent by signal
+    """
+    license_data = kwargs.get('license', None)
+    if not license_data or not isinstance(license_data, SubscriptionLicenseData):
+        logger.error("Received null or incorrect data from SUBSCRIPTION_LICENSE_MODIFIED")
+        return
+    non_pii_fields = attrs.asdict(license_data,
```

Comment:
I think we can remove `non_pii_fields` here.
1. `assigned_lms_user_id` is not considered pii.
2. If `assigned_email` were in the message, the more dangerous place for it would be Kafka. This would at least let us know. Hopefully we have dealt with PII on the publishing side.